src/main/java/org/wildfly/security/_private/ElytronMessages.java

```diff
@@ -58,32 +58,24 @@
     @Message(value = "Parse error")
     String parseError();
 
-    @Message(id = 2, value = "Invalid URI \"%s\" specified for attribute \"%s\" of element \"%s\"")
-    ElytronXMLParseException xmlInvalidUri(@Param XMLStreamReader reader, String attributeValue, String attributeName, QName elementName);
+    // id = 2
```

Comment:
Yeah.  Since we have no compatibility to maintain (yet), we can put new messages here.